internal/exec/stages/files/files.go

```diff
@@ -132,7 +132,7 @@ func (s *stage) checkRelabeling() error {
 
 	// initialize to non-nil (whereas a nil slice means not to append, even
```

Comment:
Outdated comment. I think the point it makes is still valid for `map`s, so you can just `s/slice/map/`.